otherlibs/win32unix/realpath.c

```diff
@@ -0,0 +1,79 @@
+/**************************************************************************/
+/*                                                                        */
+/*                                 OCaml                                  */
+/*                                                                        */
+/*                         The OCaml programmers                          */
+/*                                                                        */
+/*   Copyright 2020 Institut National de Recherche en Informatique et     */
+/*     en Automatique.                                                    */
+/*                                                                        */
+/*   All rights reserved.  This file is distributed under the terms of    */
+/*   the GNU Lesser General Public License version 2.1, with the          */
+/*   special exception on linking described in the file LICENSE.          */
+/*                                                                        */
+/**************************************************************************/
+
+#define CAML_INTERNALS
+
+/*
+ * Windows Vista functions enabled
+ */
+#undef _WIN32_WINNT
+#define _WIN32_WINNT 0x0600
+
+#include <caml/mlvalues.h>
+#include <caml/memory.h>
+#include <caml/alloc.h>
+#include <caml/fail.h>
+#include <caml/osdeps.h>
+#include "unixsupport.h"
+
+#include <windows.h>
+#include <fileapi.h>
+#include <stdio.h>
```

Comment:
[It seems](https://ci.appveyor.com/project/avsm/ocaml/builds/37172174/job/4y5xmsogcn7bchtt#L1957) one of the windows CI builds can't find <fileapi.h>. Can please a Windows user provide guidance on what exactly should be done here.